package.json

```diff
@@ -14,7 +14,8 @@
         "coveralls": "^3.0.2",
         "eslint": "^5.11.1",
         "eslint-config-proton": "^1.0.5",
-        "jest": "^24.9.0"
+        "jest": "^24.9.0",
+        "nodemon": "^1.19.3"
```

Comment:
Can you add `nodemon` as `devDependencies`?

There's no point in adding it as `dependencies` bec